soccer/src/soccer/processor.cpp

```diff
@@ -143,7 +145,9 @@ void Processor::run() {
         ////////////////
         // Inputs
         // TODO(#1558): Backport spin_all and use it for our main executor.
-        ros_executor_->spin_some();
+        for (int i = 0; i < 10; i++) {
+            ros_executor_->spin_some();
+        }
```

Comment:
This is a hack so we can process multiple debug drawing messages in the same frame, which will be necessary. The queue length is 10, so that's the most messages we could process at once anyway.

Pending issue #1558 this will be removed.